GitUI/UserControls/RevisionGrid/Columns/MessageColumnProvider.cs

```diff
@@ -289,38 +290,41 @@ public override void OnCellFormatting(DataGridViewCellFormattingEventArgs e, Git
 
         public override bool TryGetToolTip(DataGridViewCellMouseEventArgs e, GitRevision revision, out string toolTip)
         {
+            _toolTipBuiler.Clear();
```

Comment:
It looks like we have similar implementations in multiple places (`GitRevision` and `MessageColumnProvider`).
Can we extract the functionality into a `ToolTipBuilder` class that deals with all nuances of building a tooltip?